server/spinwick.go

```diff
@@ -140,8 +140,27 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 
 	mlog.Info("Provisioning Server - Installation request")
 
+	headers := map[string]string{
+		"x-api-key": s.Config.AWSAPIKey,
+	}
+	cloudClient := cloudModel.NewClientWithHeaders(s.Config.ProvisionerServer, headers)
+
+	var groupID string
+	var group *cloudModel.Group
+	if len(s.Config.CloudGroupID) != 0 {
+		group, err = cloudClient.GetGroup(s.Config.CloudGroupID)
+		if err != nil {
+			return request.WithError(errors.Wrapf(err, "unable to get group with ID %s", s.Config.CloudGroupID))
+		}
```

Comment:
Did the auth lambda used by Matterwick get updated to allow for hitting the `/group` API of the provisioner?

I had to remove this section from the cloud plugin and just use the group without checking for it first so that we didn't have to open the auth lambda up any further: https://github.com/mattermost/mattermost-plugin-cloud/pull/43/files

Reply:
hummm, will coment this out for now and check the auth. good catch!